### PR TITLE
Initial Graphics Test

### LINKED ISSUE #41: Game GUI
should be able to load images, have colliders for pieces on board, drag and drop pieces on board, render board and pieces, etc - probably use a graphics library like p5 or something - this will be a significant task

### PR DESCRIPTION
**Pull request by jgross11**
hello

### CODE REVIEW

**Review by robbycw — COMMENTED** [review]
Looks like a good start! It's nice being able to move the king around. Were we still thinking of eventually having configurable sizes to the board?

**Comment by jgross11** [comment]
> Looks like a good start! It's nice being able to move the king around. Were we still thinking of eventually having configurable sizes to the board?

Yep - this is mainly just to show basic functionality with P5. We'll need a very fancy system in place to handle drawing variably sized boards, detecting bounds for clicking + dragging pieces, etc. The game model will have all this info, and the view will (probably) handle the rendering of that model while also performing visual updates to it while waiting for the backend to confirm a change (ex. moving a piece to a different square)..

**Comment by jgross11** [comment]
start of #41

File: bless_this_chess/view/static/js/ViewUtils.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// thanks js
+intdiv = (a,b) => (a-a%b)/b;
```
Comment (andersonjackc) [inline bless_this_chess/view/static/js/ViewUtils.js:2]: 👀

**Comment by andersonjackc** [comment]
This is really starting to look great.  It was fun being able to capture pieces and have jumps that you wouldn't get in a normal game of chess, which is what we will eventually want anyway!

Thanks for getting work started on this again!

**Review by andersonjackc — APPROVED** [review]

**Comment by andersonjackc** [comment]
Found a bug where if you pick up your piece and then just let go it deletes itself.

File: bless_this_chess/view/static/js/ViewUtils.js (hunk 1 above)
Comment (jgross11) [reply bless_this_chess/view/static/js/ViewUtils.js:2]: 👀

**Comment by jgross11** [comment]
> Found a bug where if you pick up your piece and then just let go it deletes itself.

good catch buddy

**Pull request merged**